src/elm/Main.elm

```diff
@@ -2015,6 +1932,97 @@ update msg model =
         NoOp ->
             ( model, Cmd.none )
 
+        SchedulesResponse org repo result ->
```

Comment:
could we move this section of inbound HTTP responses into the `-- Inbound HTTP responses` section? 

they are currently placed in the `-- Other` section